src/styles/blocks/card.scss

```diff
@@ -7,7 +7,7 @@
 
   border: 1px solid #f3f3f3;
   border-radius: 5px;
-  transition-duration: 300ms;
+  transition: 300ms all;
```

Comment:
@elkhayate i meant the opposite - don't use `all` - it slows down load times
specify particular transition you're triggering